packages/artillery/lib/platform/aws/iam-cf-templates/aws-iam-lambda-cf-template.json

```diff
@@ -0,0 +1,108 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Description": "CloudFormation template to create an IAM Role for Artillery.io Lambda with attached policies",
+  "Resources": {
+    "ArtilleryDistributedTestingLambdaRole": {
+      "Type": "AWS::IAM::Role",
+      "Properties": {
+        "RoleName": "ArtilleryDistributedTestingLambdaRole",
+        "AssumeRolePolicyDocument": {
+          "Version": "2012-10-17",
+          "Statement": [
+            {
+              "Effect": "Allow",
+              "Principal": {
+                "AWS": "*"
+            },
+              "Action": "sts:AssumeRole",
+              "Condition": {}
+            }
+          ]
+        },
+        "Path": "/",
+        "Policies": [
+          {
+            "PolicyName": "ArtilleryDistributedTestingLambdaPolicy",
+            "PolicyDocument": {
+              "Version": "2012-10-17",
+              "Statement": [
+                {
+                  "Sid": "CreateOrGetLambdaRole",
+                  "Effect": "Allow",
+                  "Action": [
+                    "iam:CreateRole",
+                    "iam:GetRole",
+                    "iam:PassRole",
+                    "iam:AttachRolePolicy"
+                  ],
+                  "Resource": {"Fn::Sub": "arn:aws:iam::${AWS::AccountId}:role/artilleryio-default-lambda-role-*"}
+                },
+                {
+                  "Sid": "CreateLambdaPolicy",
+                  "Effect": "Allow",
+                  "Action": ["iam:CreatePolicy"],
+                  "Resource": {"Fn::Sub": "arn:aws:iam::${AWS::AccountId}:policy/artilleryio-lambda-policy-*"}
```

Comment:
In the original policy from the docs `Resource` here was missing `-*` at the end.